index.html

```diff
@@ -1,112 +1,117 @@
 <!DOCTYPE html>
 <html>
-  <head>
+<head>
     <meta charset="UTF-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <meta name="keywords" content="css, html5, css framework"/>
     <meta name="description" content="CSS Framework for Dummies"/>
     <link rel="stylesheet" href="tacit.min.css"/>
     <title>tacit</title>
     <link rel="shortcut icon" type="image/gif" href="logo.gif"/>
-  </head>
-  <body>
-    <section>
-      <header>
+</head>
+<body>
+<section>
```

Comment:
@smartclash I can't understand the logic. Why do you indent `header` but don't indent `section` and `body`? :)